LongWarOfTheChosen/Config/XComGameData.ini

```diff
@@ -1317,6 +1321,84 @@ DisplayEffects="LethargyEffect"
 DisplayEffects="IncreaseTimer2Effect"
 ; ExcludePlotTypes="Abandoned"
 
+[DarkEventHighAlertSitRep X2SitRepTemplate]
+MinimumForceLevel=5
+bNegativeEffect=true
+
+NegativeEffects="DarkEventHighAlertEffect"
+bExcludeFromStrategy=false
+
+[DarkEventInfiltratorSitRep X2SitRepTemplate]
+TacticalGameplayTags="DarkEvent_Infiltrator"
+MinimumForceLevel=5
+bNegativeEffect=true
+
+NegativeEffects="DarkEventInfiltratorEffect"
+bExcludeFromStrategy=false
+
+[DarkEventInfiltratorChryssalidSitRep X2SitRepTemplate]
+TacticalGameplayTags="DarkEvent_InfiltratorChryssalid"
+MinimumForceLevel=11
+bNegativeEffect=true
+
+NegativeEffects="DarkEventInfiltratorChryssalidEffect"
+bExcludeFromStrategy=false
+
+[DarkEventRapidResponseSitRep X2SitRepTemplate]
+TacticalGameplayTags="DarkEvent_RapidResponse"
+MinimumForceLevel=8
+bNegativeEffect=true
+
+NegativeEffects="DarkEventRapidResponseEffect"
+bExcludeFromStrategy=false
+
+[DarkEventReturnFireSitRep X2SitRepTemplate]
+TacticalGameplayTags="DarkEvent_ReturnFire"
+MinimumForceLevel=5
+bNegativeEffect=true
+
+NegativeEffects="DarkEventReturnFireEffect"
+bExcludeFromStrategy=false
+
+[DarkEventSealedArmorSitRep X2SitRepTemplate]
+TacticalGameplayTags="DarkEvent_SealedArmor"
+MinimumForceLevel=5
+bNegativeEffect=true
+
+NegativeEffects="DarkEventSealedArmorEffect"
+bExcludeFromStrategy=false
+
+[DarkEventUndyingLoyaltySitRep X2SitRepTemplate]
+TacticalGameplayTags="DarkEvent_UndyingLoyalty"
+MinimumForceLevel=6
+bNegativeEffect=true
+
+NegativeEffects="DarkEventUndyingLoyaltyEffect"
+bExcludeFromStrategy=false
+
+[DarkEventVigilanceSitRep X2SitRepTemplate]
+TacticalGameplayTags="DarkEvent_Vigilance"
+MinimumForceLevel=6
+bNegativeEffect=true
+
+NegativeEffects="DarkEventVigilanceEffect"
+bExcludeFromStrategy=false
+
+[TheLost X2SitRepTemplate]
++NegativeEffects="TheLostSitRepEffect"
+
+; Keep the Lost off of untimed missions
```

Comment:
Should add:  SupplyLineRaid_LW and Sabotage_LW

Reply:
Wait, what? They're both untimed, right? So they shouldn't be in the list (although the Facility mission does have a soft timer with cavalry reinforcements).

Reply:
...Nevermind, got it confused with MISSION_TYPE_SIT_REP_EXCLUSIONS

Reply:
Also, if you think changes need to be made, as in this case, don't hesitate to pick the Request Changes options when submitting the review.